### PR TITLE
update version and build

### PR DESCRIPTION
**Pull request by wholmgren**
It's my understanding that I need to do this to release the latest version of the package...

### CODE REVIEW

**Comment by wholmgren** [comment]
Not sure why this is failing now. Maybe something changed with the conda-forge configuration?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ source:
   md5: 552fcf215911b5bb7b7a2e5ad6a9a6e6
```
Comment (ocefpaf) [inline recipe/meta.yaml]: You need to update the checksum. The latest version source this is `1fdef1d50270d22f8d89d9032fca2730`.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ source:
...
 
 build:
-  number: 0
+  number: 1
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Set this to `0` as this is the first build of the new version.

**Comment by pelson** [comment]
This looks very sensible to me. I'd suggest merging this @wholmgren 👍

**Comment by wholmgren** [comment]
Ah, I forgot about this. Thanks for the ping @pelson.

**Pull request merged**